<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerSettingsForm</class>
 <widget class="QWidget" name="TimerSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>timer settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="labelMinutes">
       <property name="text">
        <string>minutes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelSeconds">
       <property name="text">
        <string>seconds</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelHours">
       <property name="text">
        <string>hours</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="sbHours"/>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="sbSeconds">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="sbMinutes">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sbHours</tabstop>
  <tabstop>sbMinutes</tabstop>
  <tabstop>sbSeconds</tabstop>
  <tabstop>pushOk</tabstop>
  <tabstop>pushCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
